.github/workflows/sanity.yaml

```
@@ -0,0 +1,32 @@
+name: sanity
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+    - main
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v3
```

Nit: I think there are newer versions of both of these actions. We should go ahead and use them (and I should have done that for the release action but I forgot :sweat_smile:).

```suggestion
      - uses: actions/checkout@v3
      - uses: actions/setup-go@v4
```

---

@everettraven done